examples/qs-only/yarn.lock

```diff
@@ -0,0 +1,5411 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This lockfile shouldn't be here since the folder is part of the monorepo.

Reply:
Fixed.